packages/desktop/electron/main.js

```diff
@@ -38,10 +56,25 @@ let lastError = {}
  */
 const handleError = (errorType, error, isRenderProcessError) => {
     if (app.isPackaged) {
+        const diagnostics = getDiagnostics()
+
         lastError = {
-            diagnostics: getDiagnostics(),
-            errorType,
+            diagnostics,
             error,
+            errorType,
+        }
+
+        if (SEND_CRASH_REPORTS) {
+            Sentry.captureException(
+                new Error(
+                    JSON.stringify({
+                        type: errorType,
+                        message: error.message || error.reason || error,
+                        stack: error.stack || undefined,
+                        diagnostics,
+                    })
+                )
+            )
```

Comment:
I think it might be better if we capture the original error and then add the error type as [context](https://docs.sentry.io/platforms/javascript/guides/electron/enriching-events/context/#passing-context-directly):

```suggestion
            Sentry.captureException(error, { type: errorType })
```

The diagnostics probably don't need to be added as they should be collected automatically by Sentry

Reply:
The stacktrace may also include the OS username, so if we capture the original error I think it should be easier for Sentry's data scrubbing to remove

Reply:
Actually I might need to investigate this further :thinking: Sentry seems to report both as "non-error" types

Reply:
I am happy to make this change. Just let me know after your investigation :mag: